.gitlab-ci.yml

```diff
@@ -1,19 +1,17 @@
-image: ocaml/opam:ubuntu
-
-# this doesn't seem to work
-cache:
-  paths:
-    - .opamcache
+image: ubuntu:latest
```

Comment:
This introduces the issue fixed by #7376

Reply:
I had realized this, but what about the rest of this PR (suppose I backport this PR and #7376 at once).

Reply:
Yeah I guess that works.
I reacted a bit too much to that issue appearing.